test/test_api/test_api.py

```diff
@@ -464,3 +465,92 @@ def test_do_dummy_prediction(dask_client, fit_dictionary_tabular):
     estimator._clean_logger()
 
     del estimator
+
+
+@unittest.mock.patch('autoPyTorch.evaluation.train_evaluator.eval_function',
+                     new=dummy_eval_function)
+@pytest.mark.parametrize('openml_id', (40981, ))
```

Comment:
Those tests can be simplified by reflecting my previous comment.

Reply:
exactly how would these tests be simplified?